lib/location_info/tokenizer_mixin.js

```diff
@@ -82,8 +82,8 @@ exports.assign = function (tokenizer) {
         attachLocationInfo(this.currentToken);
     };
 
-    tokenizer._createDoctypeToken = function (doctypeNameFirstCh) {
-        tokenizerProto._createDoctypeToken.call(this, doctypeNameFirstCh);
+    tokenizer._createDoctypeToken = function (initializeName) {
```

Comment:
Nah, just changed it to accept `initialName` - that's a bit cleaner anyway IMO 😄